lib/ret/scene.ex

```diff
@@ -21,23 +21,26 @@ defmodule Ret.Scene do
     field(:slug, SceneSlug.Type)
     field(:name, :string)
     field(:description, :string)
-    # TODO: BP account and upload tables don't exist yet.
+    # One of "unlisted", "listed", "pinned"
+    field(:state, :string)
```

Comment:
this assertion for the state field probably belongs somewhere, i don't know the idiomatic way, probably in the changeset validation

Reply:
Sure, though I was actually thinking of using an existing library for these state machine fields. Either https://github.com/joaomdmoura/machinery or https://github.com/asiniy/ecto_state_machine
Would you rather we write our own small layer, or use a library?

Reply:
library is probably fine. didn't dig into those, but it'd be probably nice to be able to use PostgreSQL's `enum` field type if possible